eclipse-collections-code-generator/src/main/resources/impl/list/mutable/synchronizedPrimitiveList.stg

```diff
@@ -415,6 +417,15 @@ public class Synchronized<name>List
             return this.getMutable<name>List().collectWithIndex(function, target);
         }
     }
+
+    @Override
+    public Mutable<name>Stack toStack()
```

Comment:
This method should be pulled up to `AbstractSynchronized<name>Collection`.

Reply:
I have it here as AbstractSynchronized<name>Collection. is not implementing Ordered<name>Iterable.